cargo/cargo_build_script.bzl

```diff
@@ -165,6 +167,10 @@ _build_script_run = rule(
         "crate_name": attr.string(
             doc = "Name of the crate associated with this build script target",
         ),
+        "links": attr.string(
+            default = "",
+            doc = "The name of the native library this crate links against.",
```

Comment:
Can you run `docs/update_docs.sh` to update the docs?

Reply:
is `default = "",` necessary?